doc/source/howtos/virt_sandbox.rst

```diff
@@ -331,39 +382,22 @@ This example shows how to do it with ``sysbench`` from the EPEL repository.
 
       sysbench --test=oltp --db-driver=mysql --mysql-engine-trx=yes --mysql-table-engine=innodb --mysql-host=127.0.0.1 --mysql-port=3307 --mysql-user=sbtest --mysql-password=sbpass --oltp-table-size=10000 prepare
 
-3. Run the benchmark on port 3307:
+#. Run the benchmark on port 3307:
```

Comment:
Use port 6033 to connect the PXC node through proxysql. Also replace  `--mysql-port=3037` with `--mysql-port=6033`